misc/drone-scripts/validate-and-serialize.sh

```diff
@@ -13,33 +13,101 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# datastore name refers storage backed by base ESX (physical ESX) where
+# ongoing builds are going to create file which helps to serialize CI run
```

Comment:
nit: `ongoing/xxx contains the directory named after the currently running CI/ build`